tools/test-harnesses/src/utils/getDocument/index.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copied from [dom-testing-library](https://github.com/testing-library/dom-testing-library/blob/main/src/helpers.ts#L21)
```

Comment:
do we still need this?

Reply:
same as above, its not exported from `@testing-library/dom`